### DISCUSSION TITLE
New dates for our open Weekly Calls

### DISCUSSION THREAD

**Discussion by leouieda** [fatiando/community#62] [Announcements]
**We have new dates and times for our Weekly Fatiando Development Calls!** Starting on 1 September 2022, the calls will alternate weekly between **Wednesday 20:00 UTC** and **Thursday 14:00 UTC** to accommodate more of our international community. 

**Everyone is welcome!** 

---

* ℹ️ More information: https://github.com/fatiando/community
* 💬 Join our chat rooms to get notified of calls: https://www.fatiando.org/contact/
* 📆 Shared Google calendar with meeting times: https://www.fatiando.org/calendar
* 🗒️ Meeting notes discussing this change: https://github.com/fatiando/community/blob/main/development-calls/2022.md#2022-08-22).

**Comment by MGomezN** [comment]
I'm closing this discussion, to keep open the announcement with the most up to date meeting times :)
